### PR TITLE
:bug: PIC-757: Suppressed length when not available

### PR DESCRIPTION
**Pull request by katie-cawthorne**

### CODE REVIEW

**Review by theDustRoom — APPROVED** [review]
👍

**Comment by theDustRoom** [comment]
Oops, filing test.

**Review by theDustRoom — CHANGES_REQUESTED** [review]
Sorry, there's a failing test

File: mappings/community/DX12340A-convictions.json [hunk 1]
```diff
@@ -496,7 +496,10 @@
           ],
           "sentence": {
             "sentenceId": "123123134",
-            "description": "Fine",
```
Comment (katie-cawthorne) [inline mappings/community/DX12340A-convictions.json]: All tests are passing locally but in CI the integration tests are timing out before they complete (tests are passing before it times out) with error `Too long with no output (exceeded 10m0s): context deadline exceeded` 🙄
I've changed the mock data that I amended, not that I can see any reason that this should cause the tests to timeout but just want to see if it passes.
I can't see what's causing the timeout issue.

**Review by theDustRoom — APPROVED** [review]
LGTM

**Pull request merged**